hip/README.md

```diff
@@ -0,0 +1,92 @@
+# HertzBeat Improvement Proposal (HIP)
+
+## What is a HIP?
+
+The HIP is a "HertzBeat Improvement Proposal" and it's the mechanism used to propose changes to the Apache HertzBeat codebases.
+
+The changes might be in terms of new features, large code refactoring, changes to APIs.
+
+In practical terms, the HIP defines a process in which developers can submit a design doc, receive feedback and get the "go ahead" to execute.
+
+### What is the goal of a HIP?
+
+There are several goals for the HIP process:
+
+1. Ensure community technical discussion of major changes to the Apache HertzBeat codebase.
+
+2. Provide clear and thorough design documentation of the proposed changes. Make sure every HertzBeat developer will have enough context to effectively perform a code review of the Pull Requests.
+
+3. Use the HIP document to serve as the baseline on which to create the documentation for the new feature.
+
+4. Have greater scrutiny to changes are affecting the public APIs (as defined below) to reduce chances of introducing breaking changes or APIs that are not expressing an ideal semantic.
+
+It is not a goal for HIP to add undue process or slow-down the development.
+
+### When is a HIP required?
+
+* Any new feature for HertzBeat (Example: New Monitor Protocol)
+* Any change to the wire protocol APIs
+* Any large code change that will touch multiple components
+* Any change to the configuration
```

Comment:
```suggestion
* Major new feature for HertzBeat (Example: support metrics push gateway, support logs monitoring)
* Major change to the wire protocol APIs
* Large code change that will touch multiple components
```
hi, hertzbeat is still in the early development stage. If every feature needs to go through the hip process, it will slow down the development progress. It is recommended to change it to major features instead of any feature or configuration changes. After the project is stable, we can consider going through the hip process for any feature.

Reply:
Good suggestion, done.